.gitignore

```diff
@@ -4,3 +4,5 @@ node_modules
 !test/files/cjs/node_modules
 /.project
 /.settings
+
+.vscode/launch.json
```

Comment:
This should be in your global gitignore since it's not a file that's part of the project.